README.md

```diff
@@ -1 +1,154 @@
-# Math-Magicians
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Math Magicians setup</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Math Magicians <a name="about-project"></a>
+
+**To Do List** "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+
+-Make simple calculations.
+-Read a random math-related quote.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+  <li><a href="#">HTML</a></li>
+  <li><a href="#">CSS</a></li>
+    <li><a href="#">React vite</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="NA">NA</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="NA">NA</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Make Math calculation**
+- **Read a random math-related quote**
+<p align="right">
+(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- comming soon
+
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+  ## 💻 Getting Started <a name="getting-started"></a>
+
+  To get a local copy up and running it in your machine, follow the next steps.
+
+  ### Prerequisites
+
+  In order to run this project you need:
+
+- Internet connection.
+- Basic knowledge of Git/Terminal.
+- Visual Studio Code Installed.
+
+### Setup
+
+Clone this repository to your desired folder
+
+### Install
+
+Install almost all dependencies.
+This one will install stylelinter:
+
+> npm install --save-dev
+> stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+### Usage
+
+To run the project, execute in live server.
+
```

Comment:
- [ ] Kindly add all the commands used to clone, install and run the project on the user's local machine, I have noticed you have used `vite`, it can be a good idea to add all those commands needed to run the project.